core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -14,13 +13,21 @@ private[data] object NonEmptyMapImpl extends NonEmptyMapInstances with Newtype2
   private[data] def unwrap[K, A](m: Type[K, A]): SortedMap[K, A] =
     m.asInstanceOf[SortedMap[K, A]]
 
-  def fromMap[K: Order, A](as: SortedMap[K, A]): Option[NonEmptyMap[K, A]] =
+  def fromMap[K, A](as: SortedMap[K, A]): Option[NonEmptyMap[K, A]] =
     if (as.nonEmpty) Option(create(as)) else None
 
-  def fromMapUnsafe[K: Order, A](m: SortedMap[K, A]): NonEmptyMap[K, A] =
+  @deprecated("Use fromMap override without Order", "2.2.0-M2")
```

Comment:
Shouldn't this (and others) read 2.2.0-M3?

Ah, the joys of long-running PRs.

Reply:
Right, @barambani noted that in #3397 but I didn't want to change @joroKr21's commits more than the merge required. I could go ahead and fix that here, though.

Reply:
easy fix, let's do it